docs/development.rst

```diff
@@ -505,463 +505,795 @@ point, with diagrams, and an alternative workflow, see `the git docs
-       equality. Repeat steps 1-3 until this is achieved. If changes have to be
-       made to the QC model they are committed to the branch where the QC PR
-       originates from.
+-----------------------------------
+What models are appropriate to add?
+-----------------------------------
+Any model added to ``stdpopsim`` should be based the **published literature**
+and a clear citation to the relevant paper(s) should be provided with the model.
+The demographic model should include, at a minimum,
+a single population with a series of population sizes changes.
+Multi-population models typically include other **demographic events**,
+such as population splits and changes in the amount of gene flow between populations.
+The values of different parameters should be specified in number of individuals
+(for population sizes) and generations (for times).
+Sometimes, you will need to convert values published in the literature
+to these units by making some assumptions on the mutation rate;
+typically the same assumptions made by the study that published the demographic model.
+
+
+Some demographic models can be **generic**, meaning that they may be
+applied to any species.
+The `stdpopsim` :ref:`sec_catalog` contains a collection of generic models that are widely used in
+developing and testing inference methods.
+We encourage contributors to suggest additional generic models,
+which they think can be helpful for methods development.
+Note that generic models should also be accompanied with a clear citation indicating
+their usefulness.
+
+---------------------------------------------
+Getting set up to add a new demographic model
+---------------------------------------------
+
+If this is your first time implementing a demographic model in ``stdpopsim``, it's a good
+idea to take some time browsing the :ref:`sec_catalog`
+to see how existing demographic models are coded and documented.
+If you have any questions or confusion about formatting or implementing demographic models, please
+don't hesitate to `open a new issue <http://github.com/popgensims/stdpopsim/issues>`_.
+We're more than happy to answer any questions and help get you up and running.
+Before you add any code, be sure to have forked the ``stdpopsim`` repository
+and cloned it locally, following the instructions in the `GitHub Workflow`_ section.
 
-********************
-Adding a new species
-********************
-To add a new species to `stdpopsim` several things are required:
 
-1. The genome definition
-2. Generation time estimate
-3. Mutation rate (per generation)
-4. Recombination rate (per generation)
-5. Characteristic population size
+All code for a species' demographic models is written in the ``demographic_models.py``
+file in that species directory ``stdpopsim/catalog/<SPECIES-ID>/``
+(where ``<SPECIES-ID>`` is the six-character identifier of the species;
+e.g., CanFam).
+If the species does not currently have any demographic model,
```

Comment:
We're not mentioning where code for generic demographic models should be added. I actually tried to look for this in the catalog and couldn't find it. Where is it?

Reply:
I think we do. See [this part](https://popsim-consortium.github.io/stdpopsim-docs/latest/catalog.html#generic-models) in the catalog docs.

Reply:
Okay. I found the two current generic models in ``stdpopsim/models.py``. I guess I can say that generic models are defined as Python classes in this file, and simply point to the two examples that are already implemented there.

Reply:
I pushed a new [commit](https://github.com/popsim-consortium/stdpopsim/pull/1397/commits/7a3afa59fa5b3ff0feb020a3aa1452a4496e4306) with some text about this, all at the bottom of the **"What models are appropriate to add?"** section.

Reply:
I'm still unsure whether we actually want people to contribute generic models with no associated species. As opposed to having wrappers for the generic models that are implemented in `msprime.Demography`. @petrelharp do you have an opinion?

Reply:
I wasn't aware of this discussion. So it might be best not to discuss this so explicitly in the docs. We can simply mention that they exist, and if someone is interested in contributing an additional generic model, they should simply open an issue on GitHub to discuss this first. How does that sound?

Reply:
Sounds fine!